docs-site/docs/docs/about-menu.md

```diff
@@ -0,0 +1,43 @@
+The about menu provides a deeper view into your instance of Polynote. 
+
+![Settings Menu](images/settings-menu.png)
+
+The left pane contains several sections to view different pieces of data about Polynote. 
+
+### About 
+The about menu contains information about your instance's version and most recent server commit. 
+
+### Hotkeys 
+The hotkeys menu contains a list of all supported keyboard shortcuts in Polynote. 
+
+### Preferences 
+The preferences menu contains two sections - _UI preferences_ and _storage_.
+
+#### UI Preferences 
+The preferences menu is where you can customize the following settings: 
+
+- **vim**: Whether VIM input mode is enabled for code cells 
+- **theme**: Light or Dark application color scheme 
+- **notifications**: Whether Polynote can send you browser notifications. 
+
+#### Storage 
+The storage menu displays key-value pairs of all the data Polynote it storing in your browser's local storage, 
+and provides easy access to wipe that data. 
+
+### Open Kernels 
+The open kernels menu contains a list of all existing kernels for existing notebooks. You can view their path, 
+status (idle, running, or dead), and start/kill/jump to all of these kernels. 
```

Comment:
I believe that "Open Kernels" may be something of a misnomer - this is actually a list of all open notebooks alongside their Kernel status. Note that a notebook is considered open if: (1) any client has it open* (2) there is a running kernel associated with it. This means that the list shows notebooks that are not open in this particular browser, as well as potentially some notebooks that no one has open (good for finding running kernels that people have forgotten about). 

Should we consider changing this to "Open Notebooks" perhaps? 

`*` the server may keep a handle open to a notebook after it's closed for a while - 30 seconds or a minute or something.

Reply:
Thanks! I think for the sake of this PR I'll refactor it to your description and leave it as `Open Kernels`, and will open an issue to update the text here as well as in the UI.